dev/workflow/rocoto/gefs_tasks.py

```diff
@@ -72,6 +72,27 @@ def _RT_stage_ic(self):
 
         return task
 
+    def gen_control_ic(self):
+        dependencies = []
+        if self.app_config.gefstype in ['near-real-time']:
+            dep_dict = {'type': 'task', 'name': f'{self.run}_stage_ic_mem000'}
+            dependencies.append(rocoto.add_dependency(dep_dict))
```

Comment:
If this will only run for `near-real-time` mode, then the `if` isn't needed.

```suggestion
        dep_dict = {'type': 'task', 'name': f'{self.run}_stage_ic_mem000'}
        dependencies.append(rocoto.add_dependency(dep_dict))